### PR TITLE
Add OAuth to nodemailer

### PR DESCRIPTION
**Pull request by ssd71**

### CODE REVIEW

File: routes/handler.js [hunk 1]
```diff
@@ -137,7 +140,8 @@ router.post("/register", function (req, res) {
             from: 'Perception 2020 Team, CETB',
             to: user.username,
             subject: 'Your password has changed',
```
Comment (ganeshpatro321) [inline routes/handler.js]: Change the subject to "Perception 2020 | Password reset"

**Comment by ganeshpatro321** [comment]
@ssd71 Have you tested the reset password thing

**Comment by ssd71** [comment]
Yes

File: routes/handler.js (hunk 1 above)
Comment (ssd71) [reply routes/handler.js]: Understood

**Comment by ganeshpatro321** [comment]
And handle the errors. It leads to crashing.

**Comment by ssd71** [comment]
I have added a preliminary check to ensure that the event ID that is passed, is actually a number. I cannot reproduce aforementioned error locally, so I cannot see how the 'undefined' query is made.

**Comment by ganeshpatro321** [comment]
Then, handle the eventId type in the handler, check whether it is defined or not, and no errors are handled.

**Comment by ganeshpatro321** [comment]
Good work

**Pull request merged**